storage/src/system/auction/detail.rs

```diff
@@ -1234,7 +1317,7 @@ where
     let bid_addr = BidAddr::from(validator_public_key.clone());
     let reservation_bid_keys = provider.get_keys_by_prefix(
         &bid_addr
-            .reservation_prefix()
+            .reserved_account_prefix()
```

Comment:
Same question about only considering the accounts that have reservations

Reply:
Same answer.